packages/core/src/components/cv-tag/cv-tag.vue

```diff
@@ -46,15 +62,33 @@ export default {
     label: { type: String, required: true },
     kind: {
       type: String,
-      default: componentsTags[0],
       validator(val) {
         return componentsTags.includes(val);
       },
     },
+    type: {
```

Comment:
So as to not re-use standard attributes elsewhere we have used kind for this type of switch.